air/src/air/context.rs

```diff
@@ -325,26 +280,6 @@ impl<B: StarkField> AirContext<B> {
             self.trace_len() / 2 + 1,
             n
         );
```

Comment:
exemption points should probably be removed as well (i.e., fixed to always be 1)